parsl/executors/high_throughput/executor.py

```diff
@@ -491,6 +492,11 @@ def connected_managers(self):
         workers = self.command_client.run("MANAGERS")
         return workers
 
+    @property
+    def block_status(self):
```

Comment:
Let's just get the `idle_duration` from the `MANAGERS` command itself. That would be simpler.